stdlib/asm/crypto/hashes/native.masm

```diff
@@ -9,7 +26,7 @@
 #! Cycles: 4 + 3 * words, where `words` is the `start_addr - end_addr - 1`
 #!
 #! Where `A` is the capacity word that will be used by the hashing function, and `B'` the hash output.
-proc.hash_even_memory_addresses
+export.hash_even_memory_addresses
```

Comment:
nit: the addresses don't have to be even - right? (it's the number of words that needs to be even?) Maybe rename this to something like `hash_memory_even`?

Reply:
yep, that is right. changed the name!